gamefixes-steam/2677660.py

```diff
@@ -0,0 +1,14 @@
+"""Game fix for Indiana Jones and the Great Circle"""
+
+from protonfixes import util
+
+
+def main() -> None:
+	"""Workarounds for various graphics driver bugs encountered in this game"""
+	# RadV: Missing textures and glowing eyes
+	util.set_environment('radv_legacy_sparse_binding', 'true')
+	util.set_environment('radv_zero_vram', 'true')
+	util.set_environment('RADV_DEBUG', 'nodcc')
+	# Nvidia: bad performance due to poor VRAM utilization
```

Comment:
For the NVIDIA driver bug, where's the relevant upstream issue? This needs to be reported upstream so if there isn't one already, please create one as a ProtonDB report will not suffice.

Reply:
https://forums.developer.nvidia.com/t/bug-indiana-jones-and-the-great-circle-only-runs-at-40-of-max-tdp-on-rtx-4090/315916/55

Reply:
Never mind. Found it:

https://forums.developer.nvidia.com/t/bug-indiana-jones-and-the-great-circle-only-runs-at-40-of-max-tdp-on-rtx-4090/315916/55